pkg/kubectl/service.go

```diff
@@ -196,5 +196,8 @@ func generate(genericParams map[string]interface{}) (runtime.Object, error) {
 			return nil, fmt.Errorf("unknown session affinity: %s", params["session-affinity"])
 		}
 	}
+	if params["headless"] == "true" {
```

Comment:
> Ah, thought you were talking about the cobra struff. I'm still fine with what @saturnism has here.

No.  The result is different.  https://golang.org/src/strconv/atob.go#L10 is used by the `GetBool` method and provides a much better mapping to intent.